### PR TITLE
Fix documentation of `MarketDestroyed` event

### PR DESCRIPTION
**Pull request by maltekliemann**

### CODE REVIEW

File: zrml/prediction-markets/src/lib.rs [hunk 1]
```diff
@@ -1267,7 +1267,7 @@ mod pallet {
         MarketApproved(MarketIdOf<T>, MarketStatus),
         /// A market has been created \[market_id, creator\]
         MarketCreated(MarketIdOf<T>, Market<T::AccountId, T::BlockNumber, MomentOf<T>>),
-        /// A market has been created \[market_id, creator\]
+        /// A market has been destroyed \[market_id, creator\]
```
Comment (Chralt98) [inline zrml/prediction-markets/src/lib.rs:1270]:
```suggestion
        /// A market has been destroyed. \[market_id\]
```

**Review by Chralt98 — COMMENTED** [review]
You had one job. :-D just kidding. Maybe we should write for every event a period after each sentence consequently. That's okay to me then.

File: zrml/prediction-markets/src/lib.rs (hunk 1 above)
Comment (maltekliemann) [reply zrml/prediction-markets/src/lib.rs]: :facepalm:

**Review by Chralt98 — APPROVED** [review]
Looks good to me 👍

**Pull request merged**